benchmarks/dynamo/huggingface_llm_models.py

```diff
@@ -0,0 +1,96 @@
+import subprocess
```

Comment:
For this new file, I think we should have this in https://github.com/pytorch/benchmark/blob/main/torchbenchmark/util/framework/huggingface/model_factory.py and bring it to CI by bumping TorchBench pinned commit.  To be fair, updating TorchBench on PyTorch CI hasn't been done for a while, and I'm reviving this process (https://github.com/pytorch/pytorch/pull/161461).  So, instead of waiting for that, this is probably ok to keep this file here for now to unblock the new models.  Then, we can add it to TorchBench.  Any thoughts?